app/controllers/account/invitations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Account::InvitationsController < Account::ApplicationController
+  include Account::Invitations::ControllerBase
+
+  private
+
+  def permitted_membership_fields
+    # We render app/views/account/memberships/_attributes.html.erb on the Invitation form
+    # so by default we permit all of the fields that the MembershipController permits. If
+    # you want more control you can remove the following line and then return an array of
+    # membership attributes that you want to allow at the time of iniviation. For example:
+    # [ :description ]
+    Account::MembershipsController.new.send(:permitted_fields)
```

Comment:
Potentially we could do this in `core` and either remove this method, or maybe even the entire file. I kinda think it makes sense to expose it here and give people an easier way to override the default.